launcher/src/main/java/org/apache/spark/launcher/ChildProcAppHandle.java

```diff
@@ -46,6 +47,25 @@ public synchronized void disconnect() {
     }
   }
 
+  /**
+   * Parses the logs of {@code spark-submit} and returns the last exception thrown.
+   *
+   * <p>
+   *   Since {@link SparkLauncher} runs {@code spark-submit} in a sub-process, its difficult to
```

Comment:
it's

also, small nit: keep the text at the same indent level as the p tags (as is done in other classes).